canonical-websites/vanillaframework.io
# Update page headers with a hero style with background image.

## Done

Updated the page headers across the site - the same style as the existing homepage.

## QA

 - `./run`
 - Check all of the top-level pages (not docs) now have new headers
 - Compare with the [design](https://app.zenhub.com/workspace/o/canonical-websites/vanillaframework.io/issues/137)

## Issue / Card

Fixes #137
